<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-PRC4QK7" failures="1" tests="2" name="Default test(failed)(failed)(failed)" time="23.491" errors="0" timestamp="2023-01-11T09:41:36 IST">
  <testcase classname="TestCases.failedTestCase" name="Test2" time="0.116">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.failedTestCase.Test2(failedTestCase.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase classname="TestCases.failedTestCase" name="Test3" time="0.062"/>
  <testcase classname="TestCases.failedTestCase" name="Test1" time="0.0">
    <ignored/>
  </testcase> <!-- Test1 -->
</testsuite> <!-- Default test(failed)(failed)(failed) -->
